ledger/sandbox/src/main/scala/com/digitalasset/platform/apiserver/ApiServerConfig.scala

```diff
@@ -22,5 +22,5 @@ case class ApiServerConfig(
     maxInboundMessageSize: Int,
     eventsPageSize: Int = IndexConfiguration.DefaultEventsPageSize,
     portFile: Option[Path],
-    seeding: Option[Seeding] // Third-party participants may need to set this to `None`.
+    seeding: Seeding
```

Comment:
```suggestion
    seeding: Seeding,
```

Had to remove the comma because of a bug in the Scala compiler which makes this an error:

```scala
def func(
  arg: Type, // comment
)
```